content/post/build-code.md

```diff
@@ -381,25 +365,24 @@ make:
 ./autogen.sh
 ```
 ```bash
-./configure --enable-silent-rules --with-lokit-path=${MASTER}/include \
-            --with-lo-path=${MASTER}/instdir \
-            --with-poco-includes=<POCOINST>/include --with-poco-libs=<POCOINST>/lib \
-            --enable-debug
+./configure --with-lokit-path=<LIBREOFFICEDIRECTORY>/include \
+            --with-lo-path=<LIBREOFFICEDIRECTORY>/instdir \
 ```
 
+where `<LIBREOFFICEDIRECTORY>` is the location of the LibreOffice source tree you have built
+in the previous steps. `
+
 You can also add extra flags to customize your build
 
-- You can also add `--disable-ssl` instead of changing coolwsd.xml everytime you want to disable ssl.
+- If you built POCO from source, add `--with-poco-includes=<POCODIRECTORY>/include --with-poco-libs=<POCODIRECTORY>/lib`
+- Add `--enable-debug` to enable debugging and link with debugging versons of POCO libraries
```

Comment:
I think and since this is a page targeting developers, being able to debug is probably a must. I love that you have added this line that explains what it does but could you still include it in the ./configure line?